client/src/components/users/Login.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from "react";
+import { userLogIn } from "../../api/users";
+import Cookies from "universal-cookie";
+import Modal from "react-modal";
+import {
+  Button,
+  Form,
+  FormGroup,
+  Label,
+  Input
+} from "reactstrap";
+
+const cookies = new Cookies();
+const customStyles = {
+  content: {
```

Comment:
I'd put these in a separate css file

Reply:
Done (plus one more constant)
Will try to clean main CSS file as well